src/main.py

```diff
@@ -233,26 +227,47 @@ def internet_on():
 		return False
 
 
-def gettoken():
-	token = mc.get("access_token")
-	refresh = config['alexa']['refresh_token']
-	if token:
-		return token
-	elif refresh:
+class Token(object):
+
+	_token = None
+	_timestamp = None
+	_validity = 3570
+
+	def __init__(self, aconfig):
+
+		self._aconfig = aconfig
+
+		self.renew()
+
+	def get(self):
+
+		if time.time() - self._timestamp > self._validity:
+			self.renew()
```

Comment:
When the token expires, you get a renew message. So you'd see an info message and a debug message? Maybe we should change the one message to be more informative?